test/metabase/http_client.clj

```diff
@@ -23,17 +24,23 @@
   "Prefix to automatically prepend to the URL of calls made with `client`."
   (str "http://localhost:" (config/config-str :mb-jetty-port) "/api/"))
 
+(def ^:dynamic *url-encode-query-parameters?*
+  "Whether to automatically URL-encode query parameter keys and values."
+  true)
```

Comment:
where in the code is this ever adjusted and why would we want to not encode?

Reply:
I'm not sure we'd ever want to not encode it. I guess you can just manually append your weird parameters to the URL itself if you don't want them to automatically be encoded. I'll remove this option since it's just clutter